### ISSUE TITLE
\\\ Hacktoberfest 2020 ///

### ISSUE THREAD

**Issue by vfdev-5** [pytorch/ignite#1257]
Hi, there ! For newbies who would like to gain some new skills in programming features in Python for a neural networks training tool, here is our project : https://github.com/pytorch/ignite which could be a friendly place to begin. We can help with the whole contributing process to drive the PRs up to merging it.
Our ["help wanted"](https://github.com/pytorch/ignite/issues?q=is%3Aissue+is%3Aopen+label%3AHacktoberfest) issues vary from small feature requests, house keeping improvements, CI up to specific neural network training features (but no fear).
Please, checkout our project's issues and if you wish to begin with an issue, just say about it there and we can discuss more. Thanks !

PS: personnally, it's my 4th year with HF and really excited about this cool event ! Hope everyone could enjoy and learn things as I could previously.

- What is Hactoberfest ? => https://hacktoberfest.digitalocean.com

**Comment by vfdev-5** [comment]
FAQ: 

> Contributions during September don't count toward Hacktoberfest. Only pull requests submitted between October 1st to 31st will count.

https://discord.com/channels/720788516837261322/735506302784110652/750804457473966191

**Comment by vfdev-5** [comment]
Helpful info from organizer's discord: https://discord.com/channels/720788516837261322/735505751665147995/750389879011147796

> If you're here you're probably looking to learn more about contributing to open source or participating in Hacktoberfest this year.

> Here are just a few of the ways Hacktoberfest will be better this year:

> 1. September is Preptember – a full month for maintainers to groom your repositories and for contributors to learn about making quality pull requests.
> 2. We’re introducing a virtual event kit to help event organizers and attendees get together remotely and safely. 
> 3. We’re offering an alternative, environmentally friendly choice of swag: You can choose the ‘Pick trees, not tees’ option to replace a Hacktoberfest T-shirt with a planted tree in celebration of your accomplishments.

> What does this look like in practice?

> For folks new to open source, or who are looking to contribute this year:
>  - Take a breath, contributing to open source for the first time can be scary. It's okay to be anxious, but as long as you follow the guidelines in a project's CONTRIBUTING.md file you're doing fine.
>   - This community is here for you. You can lean on the experienced members here by asking questions and finding cool places to start contributing.
>  - If you're looking for a good place to start, look for issues tagged "good first issue" and "hacktoberfest."
>  - If you're not comfortable with contributing code, there are still tons of issues to choose from like fixing documentation, creating new logos, improving parts of a project's UI or style consistency. 
> We’ve put together this video for folks new to opensource: http://do.co/hf_firstpr

**Comment by vfdev-5** [comment]
> https://discord.com/channels/720788516837261322/745069629415620728/753507570349506642
> 
> A nice place to learn how to contribute to any GitHub repo. Including Hacktoberfest.
> https://github.com/firstcontributions/first-contributions

Thanks @oke-aditya

**Comment by vfdev-5** [comment]
> ### Values

> Inspired by you, the community, through your actions and stories.

 >   1. Everyone is welcome! Participants in Hacktoberfest represented 151 countries and thousands of unique skill sets. This program welcomes everyone already who’s already part of the open source software community, and anyone who is interested in sectioning in.
>    2. Quantity is fun, quality is key. Participating in Hacktoberfest leads to personal growth, professional opportunities, and community building. However, it all begins with meaningful contributions to open source software.
>    3. Short-term action, long-term impact. In the open source software community, we are standing on the shoulders of those who came before us. Your participation has a lasting effect on people and technology long after October. This is a voyage not a race.

See more:  https://hacktoberfest.digitalocean.com/details

**Comment by vfdev-5** [comment]
Rules are updated. Ignite has topic "hacktoberfest" and all merged PR will count for 5 Hacktoberfest PRs:
```python
PRs count if:
Submitted during the month of October AND (
  The PR is labelled as hacktoberfest-accepted by a maintainer OR
  Submitted in a repo with the hacktoberfest topic AND (
    The PR is merged OR
    The PR has been approved
  )
) 
``` 

https://hacktoberfest.digitalocean.com/hacktoberfest-update

**Comment by vfdev-5** [comment]
🎉 It was a great month with a lot of open-source contributions from Hacktoberfest 2020 folks. We are very grateful to all contributors for working on the project ! 🎉 

🙏 Thanks to @harsh8398, @thescripted, @RaviTezu, @vcarpani, @kamahori and others !
In addition we would like particularly acknowledge outstanding contributions of:
- Anton Grübel, @gruebel
- Taras Savchyn, @trsvchn
 
Thanks you all and hope to see more of your contributions later !
